.gitignore

```diff
@@ -51,7 +51,6 @@ ignore/
 data/austraits/
 
 #
-*.pdf
 *.svg
 inst/cheatsheet/austraits-cheatsheet.pptx
```

Comment:
why exclude? shouldn't we anyway stop a copy of the ppt as the original?

Reply:
Good point! so everyone has a copy